server/app/routes/barmen.js

```diff
@@ -11,4 +11,8 @@ router.get('/:id(\\d+)', guard.check('barman:read'), am(barmanController.getBarm
 router.put('/:id(\\d+)', guard.check('barman:write'), am(barmanController.updateBarman));
 router.delete('/:id(\\d+)', guard.check('barman:write'), am(barmanController.deleteBarman));
 
+router.get('/:id(\\d+)/services', guard.check('barman:read', 'service:read'), am(barmanController.getServicesBarman));
```

Comment:
No permissions for now, otherwise, nobody will be able to use it :smile: 
I will do it with #34